### PR TITLE
Migrate up to ACL if there are no warning

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

File: cmd/lakefs/cmd/migrate.go [hunk 1]
```diff
@@ -94,34 +94,41 @@ func migrateToACL(ctx context.Context, kvStore kv.Store, cfg *config.Config, log
...
 			}
-			fmt.Println(w.Warn)
 		}
 		fmt.Println()
 	}
 	for _, username := range usersWithPolicies {
-		fmt.Printf("USER (%s)  detaching directly-attached policies\n", username)
+		if printMessages {
+			fmt.Printf("USER (%s)  detaching directly-attached policies\n", username)
+		}
```
Comment (arielshaqed) [inline cmd/lakefs/cmd/migrate.go:129]: I still don't understand the requirement: why are these treated specially?  Can't we just put these on warnings?

**Review by arielshaqed — APPROVED** [review]
Code implementation of logic seems a bit awkward.  I think we can simplify it with any or all of the following:

* Use warnings to report users with directly-attached policies, rather than giving them special treatment.
* _Always_ run `migrateToACL` with `printMessages=true`, then re-run it with printMessages=false if we need to (because there are no warnings, **or** because force was applied).

THANKS!

**Comment by guy-har** [comment]
@arielshaqed, 
according to your comments:
> Always run migrateToACL with printMessages=true, then re-run it with printMessages=false if we need to (because there are no warnings, or because force was applied).
If we do that force will not print warnings, which was implemented that way, I didn't want to change that behavior 

> Use warnings to report users with directly-attached policies, rather than giving them special treatment.
Considered that, but it doesn't exactly fit for warnings, it has no ACL and different behavior than warnings

Also noting that we will not need to maintain this code, the migration code should be removed in the following lakeFS version.

File: cmd/lakefs/cmd/migrate.go (hunk 1 above)
Comment (guy-har) [reply cmd/lakefs/cmd/migrate.go:129]: Currently, warnings have an ACL-specific logic on printing. Tried using warnings at first which resulted, in making them fit both group warnings and user-policy warnings felt like over-complicating things, but looks like I ended up doing that anyway.

**Pull request merged**